components/swatch/stories/template.js

```diff
@@ -114,6 +130,13 @@ export const Template = ({
 								}[size] || "100"),
 								useRef: false,
 							}, context)] : []),
+							...(isAddSwatch ? [Icon({
+								customClasses: [`${rootClass}-icon`],
+								setName: "workflow",
+								size,
+								iconName: "Add",
```

Comment:
Could you make a ticket to follow up on this icon? Figma is calling for the Add UI icon (not the Add workflow icon), but it is part of that group of icons that's missing from our UI package right now. Larz is working on it though!

I just don't want us to forget to eventually use the correct icon. I can share the thread with you in Slack if you want to add it in the the follow-up ticket, too.